clippy_lints/src/lib.rs

```diff
@@ -964,7 +964,9 @@ pub fn register_plugins(store: &mut rustc_lint::LintStore, sess: &Session, conf:
     store.register_late_pass(|| box strings::StringAdd);
     store.register_late_pass(|| box implicit_return::ImplicitReturn);
     store.register_late_pass(|| box implicit_saturating_sub::ImplicitSaturatingSub);
-    store.register_late_pass(|| box methods::Methods);
+    let parsed_msrv = conf.msrv.as_ref().and_then(|msrv| parse_msrv(msrv, None, None));
```

Comment:
Why are all of these `as_ref` necessary, if you clone the `Version` anyway?

Reply:
Yeah I think we can remove these, they're unnecessary